index.html

```diff
@@ -21,15 +21,15 @@
 </header>
 <section>
     <div class="fitem">
-    <h1>I'M Saba <br> Glad to see <br> you!</h1>
+    <h1>I'M Saba Ahmad <br> Glad to see <br> you!</h1>
     <p>
         I'm a software developer! I can help you <br> build a product , feature or website look <br> through some of my work <br> experience ! if you like what you see and <br> have a project you need coded, don't <br> hesitate to contact me
     </p>
 </div>
 </section>
 <footer>
     <div class="footer-content">
-      <h3>LET'S CONNECT</h3>
+      <h3>LETS CONNECT</h3>
       <ul class="socials">
         <li><a href="#"><i class="fa fa-facebook"></i></a></li>
         <li><a href="#"><i class="fa fa-twitter"></i></a></li>
```

Comment:
- [ ] Kindly make sure to not use `footer` because this is normally used on the last part of the page and there are other many parts to be added to your portfolio which we will cover next week before reaching to the footer. Please make to pull all social media into that section.